packages/fxa-auth-server/lib/senders/emails/templates/inactiveAccountFirstWarning/index.mjml

```diff
@@ -0,0 +1,33 @@
+<%# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. %>
+
+<mj-section>
+  <mj-column>
+    <mj-text css-class="text-header">
+      <span data-l10n-id="inactiveAccountFirstWarning-title">Do you want to keep your Mozilla account and data?</span>
+    </mj-text>
+  </mj-column>
+</mj-section>
+
+<mj-section>
+  <mj-column>
+    <mj-text css-class="text-sub-body">
+      <span data-l10n-id="inactiveAccountFirstWarning-account-description">Your Mozilla account is used to access free privacy and browsing products like Firefox Sync, Monitor, Relay, and MDN.</span>
+    </mj-text>
+    <mj-text css-class="text-sub-body">
+      <span data-l10n-id="inactiveAccountFirstWarning-inactive-status">We noticed you haven’t signed in for 2 years.</span>
+    </mj-text>
+    <mj-text css-class="text-sub-body">
+      <span data-l10n-id="inactiveAccountFirstWarning-impact" data-l10n-args="<%= JSON.stringify({deletionDate}) %>">Your account and your personal data will be permanently deleted on <strong>{ $deletionDate }</strong> because you haven’t been active.</span>
```

Comment:
Self-review: This PR is mostly ready for review but I want to look into proper localization handling for dates in emails before officially marking it as ready for review.

Reply:
@vbudhram Typically we use moment.js via `constructLocalTimeString` in the handler to localize dates. That works pretty well for standalone dates as they are used in `userInfo` blocks, but I'm wondering if we might be better off using Fluent's [DATETIME](https://projectfluent.org/fluent/guide/functions.html#datetime) to allow localizers to choose the correct date format to fit in the sentence structure.

Reply:
I would probably lean towards using the `constructLocalTimeString` since we use it everywhere else and file follow up to evalulate or convert to using `DATETIME`.